agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
> As a general tendency, if something is postponed then it won't be done at all :(

I have to admit that you are right @tolusha :-) I was hoping in some softness since that's an improvement of an improvement I added to the codebase (the bats files). But man you don't seem to be much indulgent today so let's try to fix that ;-) 
Let me try to address that today. If takes too long I will ask you again to be indulgent hoping to have more luck the second time.
 
> May be I am mistaken but could you clarify what issue you faced here?

In the [fabric8 docker-maven-plugin](https://github.com/fabric8io/docker-maven-plugin) I've seen a `docker:start` goal but not a `docker:exec`. And in the tests I've written we first start a container (`docker run`) and then executes the tests inside the container (`docker exec`). Maybe `docker:watch` may help but not sure.